java/src/jmri/jmrix/AbstractConnectionConfig.java

```diff
@@ -213,6 +213,7 @@ protected int addStandardDetails(PortAdapter adapter, boolean incAdvanced, int i
      */
     @Override
     public void register() {
+        this.setInstance();
```

Comment:
I don't know if this needed. It may cause this exception to be thrown:
```
2015-11-27 09:58:45,502 jmrix.AbstractSerialConnectionConfig  ERROR - Unexpected call to setInstance [AWT-EventQueue-0]
java.lang.Exception
	at jmri.jmrix.loconet.locormi.ConnectionConfig.setInstance(ConnectionConfig.java:79)
	at jmri.jmrix.AbstractConnectionConfig.register(AbstractConnectionConfig.java:216)
	at jmri.jmrix.JmrixConfigPane.selection(JmrixConfigPane.java:339)
```
The method that calls that is:
```
protected void setInstance() {
        log.error("Unexpected call to setInstance");
        new Exception().printStackTrace();
    }
```
and appears to be benign.